zfs-dkms/PKGBUILD

```diff
@@ -35,6 +38,9 @@ prepare() {
     # fix compilation on kernel 5.5 https://github.com/openzfs/zfs/issues/9745
     patch -p1 -i ../2fcab8795c7c493845bfa277d44bc443802000b8.patch
 
+    # fix compilation on kernel 5.6 https://github.com/openzfs/zfs/pull/9961#issuecomment-591076133
+    patch -p1 -i ../0002-fix-5.6-build.patch
```

Comment:
This file is in plain diff format -- it doesn't have git commit header(s) so it is difficult to trace its provenance. Please:

- identify each upstream patch this corresponds to
- backport it onto the 0.8.3 tag, resolving conflicts if needed, I recommend using `git cherry-pick -x $sha1`
- generate the .patch using e.g. `git format-patch -2 --stdout > zfs-fix-5.6-build.patch` (or however many patches there are)

This will help me review and confirm the changes.

If the patch(es) backport cleanly and you don't need to resolve conflicts, it's also possible to just list them in the source as `https://github.com/openzfs/zfs/commit/$sha1.patch`. I prefer this if there's only one patch, but I'm not picky about it if there are multiple patches and one combined patch is more convenient. The important thing is to attribute the upstream commit(s) using git format-patch.

Reply:
Done, I had some merge conflict though. Would you like me to rebase the PR and squash everything into one commit?

Reply:
Yes, please do squash it and keep the first commit message.